keyform/templates/keyform/emails/status_update.html

```diff
@@ -0,0 +1,422 @@
+            }
+            .ExternalClass,
+            .ExternalClass p,
+            .ExternalClass span,
+            .ExternalClass font,
+            .ExternalClass td,
+            .ExternalClass div {
+                line-height: 100%;
+            }
+            .apple-link a {
+                color: inherit !important;
+                font-family: inherit !important;
+                font-size: inherit !important;
+                font-weight: inherit !important;
+                line-height: inherit !important;
+                text-decoration: none !important;
+            }
+            .btn-primary table td:hover {
+                background-color: #34495e !important;
+            }
+            .btn-primary a:hover {
+                background-color: #34495e !important;
+                border-color: #34495e !important;
+            }
+        }
+    </style>
+</head>
+
+<body>
+    <table border="0" cellpadding="0" cellspacing="0" class="body">
+        <tr>
+            <td>&nbsp;</td>
+            <td class="container">
+            	<div class="content">
+
+                    <!-- START CENTERED WHITE CONTAINER -->
+                    <span class="preheader">{% trans 'A request has been updated on Keyform' %}</span>
+                    <table class="main">
+
+                        <!-- START MAIN CONTENT AREA -->
+                        <tr>
+                            <td class="wrapper">
+                                <table border="0" cellpadding="0" cellspacing="0">
+                                    <tr>
+                                        <td>
+                                            <p>{% trans 'Information for the Key Request' %}:</p>
+                                            <table border="0" cellpadding="0" cellspacing="0" class="info-table">
+                                                <tbody>
+                                                    <tr>
+                                                        <td align="left">
+                                                            <table border="0" cellpadding="3" cellspacing="1">
+                                                                <tbody>
+                                                                    <tr>
+                                                                        <td>{% trans 'Residence Hall' %}:</td>
+                                                                        <td>{{ request.building }}</td>
+                                                                    </tr>
+                                                                    <tr>
+                                                                        <td>{% trans 'Room' %} #'s:</td>
+                                                                        {% for key in request.keydata_set.all %}
+                                                                            <td>{{ key.room_number }}</td>
```

Comment:
All these keydata loops should produce a comma-delimited list instead of a <td> per item